jormungandr/src/rest/mod.rs

```diff
@@ -23,6 +24,7 @@ pub struct Context {
     pub transaction_task: Arc<Mutex<MessageBox<TransactionMsg>>>,
     pub logs: Arc<Mutex<Logs>>,
     pub server: Arc<RwLock<Option<Server>>>,
+    pub enclave: Enclave,
```

Comment:
not a big fan of this for fencing purpose, but for now it will do (and can be properly fenced later)